security/pkg/k8s/chiron/utils.go

```diff
@@ -137,6 +135,13 @@ func SignCSRK8s(client clientset.Interface, csrData []byte, signerName string, u
 	return certChain, caCert, err
 }
 
+func checkUseV1(signerName string, usages []certv1.KeyUsage) bool {
+	if len(usages) > 0 && len(signerName) > 0 && signerName != KubernatesBuiltinSignerName {
```

Comment:
The original logic is to try v1 api first and if error, then swith to v1beta1.  So we should not change the version check logic here.

Reply:
That's make sense, thanks, I think we can close this PR because the original checking logic is correct :)